stock_operating_unit/tests/test_stock_operating_unit.py

```diff
@@ -19,10 +21,20 @@ def setUp(self):
         self.ou1 = self.env.ref('operating_unit.main_operating_unit')
         # B2C Operating Unit
         self.b2c = self.env.ref('operating_unit.b2c_operating_unit')
-        # Products
-        self.product1 = self.env.ref('product.product_product_7')
-        self.product2 = self.env.ref('product.product_product_9')
-        self.product3 = self.env.ref('product.product_product_11')
+
+        # Products. Other modules set an operating unit on the product
+        # taking it from the operating unit of the user. We don't want to
+        # depend on those other modules, thus we work-around that feature
+        # by setting (for a moment) the default operating unit of the user
+        # to be empty while creating the product.
+        self.product_categ = self.ProductCategObj.create({
```

Comment:
I see this workaround, but why it was a problem creating a product with the operating unit of the user?
That the user running the tests cannot access to it?

Reply:
This test assumes certain operating units (OUs) are on certain products. When ```product_operating_unit``` was installed, that assumption was broken and the test failed.